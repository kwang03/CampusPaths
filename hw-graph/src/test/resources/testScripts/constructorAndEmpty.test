#Tests the creation/initialization of a graph and the empty behavior of a graph
CreateGraph myGraph

#Lists the nodes in empty graph
ListNodes myGraph