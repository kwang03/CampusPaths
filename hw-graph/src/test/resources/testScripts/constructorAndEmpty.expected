#Tests the creation/initialization of a graph and the empty behavior of a graph
created graph myGraph

#Lists the nodes in empty graph
myGraph contains: